packages/blade/src/components/Box/Box.tsx

```diff
@@ -107,6 +107,26 @@ const makeBoxProps = (props: BoxProps): KeysRequired<Omit<BoxProps, 'testID' | '
 
     // Visual
     backgroundColor: props.backgroundColor,
+    borderWidth: props.borderWidth,
+    borderStyle: props.borderStyle,
```

Comment:
we can choose to not expose this as well right? can just keep `solid` by default.